src/jdk.hotspot.agent/share/native/libsaproc/ps_core_common.c

```diff
@@ -300,7 +300,7 @@ bool init_classsharing_workaround(struct ps_prochandle* ph) {
       jvm_name = lib->name;
       useSharedSpacesAddr = lookup_symbol(ph, jvm_name, USE_SHARED_SPACES_SYM);
       if (useSharedSpacesAddr == 0) {
-        print_debug("can't lookup 'UseSharedSpaces' flag\n");
+        print_debug("can't lookup 'UseSharedSpaces' symbol\n");
```

Comment:
Maybe the `print_debug` at line 311 should also be updated from "flag" to "symbol"?

Reply:
fixed.  Thanks for pointing it out.